pytorch_lightning/loops/fit_loop.py

```diff
@@ -178,24 +178,25 @@ def skip(self) -> bool:
         # until `on_run_start`, we use `limit_train_batches` instead
         return self.done or self.trainer.limit_train_batches == 0
 
-    def connect(self, epoch_loop: TrainingEpochLoop):
+    def connect(self, epoch_loop: Optional[TrainingEpochLoop] = None) -> None:  # type: ignore[override]
```

Comment:
disagree. Calling connect() without arguments makes no sense. the epoch_loop is required.